Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLAutoProcess.py

```diff
@@ -316,166 +382,221 @@ def PyExec(self):
 
     def reduce(self, i):
```

Comment:
This method was long already, now it's even longer due to the contexts. Can you split this into pieces? Like processAbsorber, processTransmission, ... processSample?

Reply:
As you suggested, I splitted `reduce()` in small pieces and removed it. These individual reduction methods (`processTransmission()`, `processAbsorber()`...) are now direclty called in `PyExec`.